FluentTerminal.SystemTray/Services/TerminalsManager.cs

```diff
@@ -1,117 +1,110 @@
-        {
-            DisplayOutputRequested?.Invoke(this, new DisplayTerminalOutputRequest
-            {
-                TerminalId = terminalId,
-                Output = output
-            });
-        }
-
-        public CreateTerminalResponse CreateTerminal(CreateTerminalRequest request)
-        {
-            if (_terminals.ContainsKey(request.Id))
-            {
-                // App terminated without cleaning up, removing orphaned sessions
-                foreach (var item in _terminals.Values)
-                {
-                    item.Dispose();
-                }
-                _terminals.Clear();
-            }
-
-            ITerminalSession terminal = null;
-            try
-            {
-                if (request.SessionType == SessionType.WinPty)
-                {
-                    terminal = new WinPtySession();
-                }
-                else if (request.SessionType == SessionType.ConPty)
-                {
-                    terminal = new ConPtySession();
-                }
-                terminal.Start(request, this);
-            }
-            catch (Exception e)
-            {
-                return new CreateTerminalResponse { Error = e.ToString() };
-            }
-
-            terminal.ConnectionClosed += OnTerminalConnectionClosed;
-            _terminals.Add(terminal.Id, terminal);
-            return new CreateTerminalResponse
-            {
-                Success = true,
-                ShellExecutableName = terminal.ShellExecutableName
-            };
-        }
-
-        public void Write(int id, byte[] data)
-        {
-            if (_terminals.TryGetValue(id, out ITerminalSession terminal))
-            {
-                terminal.Write(data);
-            }
-        }
-
-        public void ResizeTerminal(int id, TerminalSize size)
-        {
-            if (_terminals.TryGetValue(id, out ITerminalSession terminal))
-            {
-                terminal.Resize(size);
-            }
-            else
-            {
-                Debug.WriteLine($"ResizeTerminal: terminal with id '{id}' was not found");
-            }
-        }
-
-        public void CloseTerminal(int id)
-        {
-            if (_terminals.TryGetValue(id, out ITerminalSession terminal))
-            {
-                _terminals.Remove(terminal.Id);
-                terminal.Close();
-            }
-        }
-
-        public string GetDefaultEnvironmentVariableString(Dictionary<string, string> additionalVariables)
-        {
-            var environmentVariables = Environment.GetEnvironmentVariables();
-            environmentVariables["TERM"] = "xterm-256color";
-            environmentVariables["TERM_PROGRAM"] = "FluentTerminal";
-            environmentVariables["TERM_PROGRAM_VERSION"] = $"{Package.Current.Id.Version.Major}.{Package.Current.Id.Version.Minor}.{Package.Current.Id.Version.Build}.{Package.Current.Id.Version.Revision}";
-
+﻿using FluentTerminal.Models;

```

Comment:
did you change your newlines to lf? I think we should stick to crlf to avoid this in the future

Reply:
In my case git is configured to "check out as-is" and "commit as-is", meaning that I haven't changed line endings. 
But I've also noticed that someone in our fork (probably @MaxRis ?) has configuration "check-out CRLF" and "commit LF", which is the default git configuration of git in Windows, and which caused some headaches to me also.
Since I've started work from our fork, chances are that I've retained some LF files.
You should raise this in Slack so that everyone adopts.

Reply:
yeah, maybe a .editorconfig might help

Reply:
Again, please share this info in Slack because it isn't my git.

Reply:
@peske I have in global git config `autocrlf = false`, it shouldn't be `git`.
I'm working in Visual Studio 2017 Community and noticed that it always seems replaces edited lines with "CRLF". One or more times, seems, I've committed changes with "CRLF" line endings, but later tried to revert back to LF in separate commits.
To avoid above issue with VS I'm keeping `.editorconfig` file locally and VS always sets LF line endings.
But strange thing seems that some .cs files in the project have all lines with CRLF endings, but others files have LF endings. Should we standardize all files to LF line endings? cc: @felixse

Reply:
@felixse mentioned in the first comment here that he wants CRLF. Because of such issues, I don't rely on VS git extension but do commits manually. In my case it is "as-is". If the file was CRLF - I'm committing CRLF. If the file was LF - I'm committing LF. This way I avoid causing problems to others.
And I believe that this is the only reasonable config - not let git to mess up...

Reply:
rather CRLF, thats the default on Windows

Reply:
All the files created by VS are CRLF anyway, so disabling any _smartness_ of git / VS will be enough.